### PR TITLE
SSN/FN fixes to 1990e and 5490

### PR DESCRIPTION
**Pull request by raquelromano-gov**
Closes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1744

image

image

### CODE REVIEW

**Comment by raquelromano-gov** [comment]
Note that there is a test in 
`test/edu-benefits/definitions/veteranId.unit.spec.jsx` that sets the no-SSN box and verifies the FN input is shown and the SSN is no longer required.

That test is sort of duplicated in `test/edu-benefits/1990e/config/sponsorVeteran.unit.spec.jsx` and in `test/edu-benefits/1995/config/veteranInformation.unit.spec.jsx`

We can also add it to `test/edu-benefits/5490/config/sponsorInformation.unit.spec.jsx` or we can remove it from the above 2 tests and rely on the definitions test. Thoughts?

File: src/js/edu-benefits/1990e/config/form.js [hunk 1]
```diff
@@ -117,11 +117,14 @@ const formConfig = {
...
             }),
             'view:veteranId': _.merge(veteranId.uiSchema, {
-              vaFileNumber: {
-                'ui:title': 'Sponsor file number',
-              },
               veteranSocialSecurityNumber: {
                 'ui:title': 'Sponsor Social Security number'
+              },
+              'view:noSSN': {
+                'ui:title': 'Sponsor doesn\'t have a Social Security number',
```
Comment (cvalarida) [inline src/js/edu-benefits/1990e/config/form.js]: Should this be "I don’t know my sponsor’s Social Security number" (from your comment [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1744#issuecomment-288191320))?

File: src/js/edu-benefits/5490/config/form.js [hunk 2]
```diff
@@ -321,22 +322,29 @@ const formConfig = {
...
               }
             }),
-            veteranSocialSecurityNumber: _.assign(ssn.uiSchema, {
-              'ui:title': 'Sponsor Social Security number'
+            'view:veteranId': _.merge(veteranId.uiSchema, {
+              veteranSocialSecurityNumber: {
+                'ui:title': 'Sponsor Social Security number'
+              },
+              'view:noSSN': {
+                'ui:title': 'Sponsor doesn\'t have a Social Security number',
```
Comment (cvalarida) [inline src/js/edu-benefits/5490/config/form.js]: Same here.

**Review by cvalarida — APPROVED** [review]

File: src/js/edu-benefits/1990e/config/form.js (hunk 1 above)
Comment (raquelromano-gov) [inline src/js/edu-benefits/1990e/config/form.js]: Yes! Done.

**Comment by raquelromano-gov** [comment]
Thoughts on unit test coverage/duplication? (see comment above)

**Comment by cvalarida** [comment]
Ah, sorry. Didn't see that when I did my review; must have skipped right over it.

Yeah...I think we can take it out of those two. I think that's what we've been doing. Of course, I can't remember where right now, but I vaguely remember @jbalboni telling me that when I duplicated a test much like this.

As long as it uses the same chunk of code in both places, I don't think we need to add the time to running unit tests.

**Comment by raquelromano-gov** [comment]
OK, SG! Merging this, and will separately pull out the other tests.

**Pull request merged**